### DISCUSSION TITLE
Galaxy Brain achievement

### DISCUSSION THREAD

**Discussion by Lzheighton** [csthenry/MagicLightAssistant#10] [Q&A] [answered]
Does the achievement mean:
"You need to open 2 discussions that are answered"
or
"You need to become the marked answer to 2 discussions?"

Or it's another different definition?

**Comment by csthenry** [comment, marked as answer]
We currently assume it's
"You need to become the marked answer to 2 discussions?"
but it seems like there are more conditions, for example you don't seem to be able to mark yourself as the answer.

Buuut I invite you to select this as the answer, because then I should have answered 2 discussions (which I didn't create myself) and be marked by the OP as the answer, soooo that should yield Galaxy Brain, hopefully.
